motr/st/utils/motr_conf_update_st.sh

```diff
@@ -176,7 +176,7 @@ main()
 	io_ops $object_id2 $block_size $block_count "$src_file" "$dest_file2"  &
 	pid=$!
 	# Let client start
-	sleep 10
+	sleep 2
```

Comment:
I do not like this change since I do not know how it affects the functionality. Why was it kept 10s to begin with ?
For now I will approve this PR but need to understand the effects of this change to make the ST work.

Reply:
I think that because the invariant checks in extmap were disabled, the writes/reads are happening faster than what the ST expects.
If the invariant checks in extmap are re-enabled, then the ST passes with sleep of 10s.
But will need more time to analyse and debug in detail.